lib/foreman_maintain/package_manager/dnf.rb

```diff
@@ -9,6 +9,24 @@ def version_locking_supported?
       true
     end
 
+    def module_enabled?(name)
+      _status, result = info(name)
+      result.match?(/Stream.+\[e\].+/)
+    end
+
+    def enable_module(name)
+      dnf_action('module enable', name, assumeyes: true)
+    end
+
+    def module_exists?(name)
+      status, _result = info(name)
+      status
```

Comment:
```suggestion
      status == 0
```

with this change the (non-) existence of the module is checked correctly for me